images/README.md

```diff
@@ -37,10 +37,10 @@ There are additional variables that may be set that affect the behavior of speci
 
 If packer build is run without specifying which images to build, then it will attempt to build all configured images. `packer inspect packer.json` will list the configured builders. The `--only` option can be specified when running `packer build` to limit the images built.
 
-For example, to build only the Ubuntu image:
+For example, to build only the Ubuntu Bionic image:
```

Comment:
Maybe we should add the version number too in this example?

Reply:
Maybe, but we do discuss it in another section. It might be good to see if we can get some copy editing on our docs to improve the overall experience.